core/src/main/java/org/apache/gravitino/lock/LockManager.java

```diff
@@ -161,12 +161,12 @@ private void startNodeCleaner() {
     lockCleaner.scheduleAtFixedRate(
         () -> {
           long nodeCount = totalNodeCount.get();
-          LOG.info("Total tree lock node count: {}", nodeCount);
+          LOG.debug("Total tree lock node count: {}", nodeCount);
           // If the total node count is greater than the maxTreeNodeInMemory * 0.5, we will do the
           // clear up in case of the memory explosion.
           if (nodeCount > maxTreeNodeInMemory * 0.5) {
             StopWatch watch = StopWatch.createStarted();
-            LOG.trace("Start to clean up the stale tree lock nodes...");
+            LOG.info("Start to clean up the stale tree lock nodes...");
```

Comment:
It it ok to use `info`?

Reply:
This line will be printed only when `nodeCount > maxTreeNodeInMemory * 0.5`; in this case, we did need to give the user some information.

Reply:
ok.